src/Pos/Wallet/Web/Api.hs

```diff
@@ -241,6 +241,15 @@ type NewPayment =
     :> Capture "amount" Coin
     :> WRes Post CTx
 
+type TxFee =
+       "txs"
+    :> "fee"
+    :> DCQueryParam "passphrase" CPassPhrase
```

Comment:
@pva701 we sure don't need it. We can create tx right away, we can not sign it w/o password. But signature length is always same, so we just need to add it to size of tx itself.

BTW, this is interesting point. I believe tx fee should use `s = tx.size + witness.size + signature.size`